docs/reference.md

```diff
@@ -0,0 +1,4 @@
+# Reference
+
+```@index
+```
```

Comment:
```suggestion
	  # Reference
	  
	  ## Contents
	  
	  ```@contents
	  Pages = ["reference.md"]
	  ```
	  
	  ## Index
	  
	  ```@index
	  Pages = ["reference.md"]
	  ```
	  
	  ```@autodocs
	  Modules = [DerivativeFreeSolvers]
	  ```
```
Taken from DCISolver. Due to the use of triple backticks, this is incorrectly indented, so it needs to be changed manually

Reply:
Not understanding what’s incorrectly indented.

Reply:
My suggestion is incorrectly indented.